client/src/slices/draftsSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { fetchUser } from '../actions';
+import { Review } from '../types';
+
+export type DraftState = Review[];
+
+const initialState: Review[] = [];
+
+const draftsSlice = createSlice({
+  name: 'drafts',
+  initialState,
+  reducers: {
```

Comment:
`reducers` will contain any reducer functions for actions that **belong to this slice**. Those are actions which only this slice of state listens for. Defining a property function here allows it to automatically be exported as an action creator.